plat/allwinner/sun50i_a64/sunxi_power.c

```diff
@@ -76,15 +81,66 @@ void sunxi_turn_off_soc(uint16_t socid)
 	}
 }
 
+static int rsb_init(void)
+{
+	int ret;
+
+	ret = rsb_init_controller();
+	if (ret)
+		return ret;
+
+	ret = rsb_set_bus_speed(SUNXI_OSC24M_CLK_IN_HZ, 400000);
+	if (ret)
+		return ret;
+
+	ret = rsb_set_device_mode(0x7c3e00);
+	if (ret)
+		return ret;
+
+	ret = rsb_set_bus_speed(SUNXI_OSC24M_CLK_IN_HZ, 3000000);
+	if (ret)
+		return ret;
+
+	return rsb_assign_runtime_address(AXP803_HW_ADDR,
+					  AXP803_RT_ADDR);
+}
+
+static int axp_setbits_8(uint8_t reg, uint8_t set_mask)
```

Comment:
You don't use `_8` for `axp_write` later, and the registers are always 8 bits anyway (unlike MMIO where the size could vary), so I think it makes sense to drop the `_8` here as well.